app/src/lib/stores/app-store.ts

```diff
@@ -1937,15 +1937,22 @@ export class AppStore extends TypedBaseStore<IAppState> {
     this._updateCurrentPullRequest(repository)
     this.updateMenuItemLabels(repository)
     this._initializeCompare(repository)
-    this.refreshIndicatorsForRepositories([repository])
+    this.refreshIndicatorsForRepositories([repository], false)
```

Comment:
Without doing this check we can end up in recursion hell due to this loop when updating the repository indicators: `_fetch -> performFetch -> _refreshRepository -> refreshIndicatorsForRepositories -> refreshIndicatorForRepository -> _fetch -> 🔁`